happi/client.py

```diff
@@ -584,6 +584,9 @@ def _store(self, device, insert=False):
         self._validate_device(device)
         # Grab information from device
         post = device.post()
+        # save the old name in case the user is trying to
+        # change the 'name' of an entry
+        the_old_name = post.get('_id', None)
```

Comment:
```suggestion
        the_old_name = post.get(self._id_key, None)
```

Reply:
Thanks @klauer  for the suggestion. 
The `post.get(self_id_key)` is already the `new_name` here...  so I don't think I can rely on this one, unless I make other changes maybe, or unless i can get it from somewhere else?

Reply:
Sorry, this suggestion shouldn't have been added. Please ignore it.